core/templates/domain/topic/learner-topic-summary.model.ts

```diff
@@ -26,6 +27,7 @@ export interface LearnerTopicSummaryBackendDict {
   'description': string;
   'version': number;
   'story_titles': string[];
+  'canonical_story_dicts': StorySummaryBackendDict[];
```

Comment:
Here and else where, should we call this "canonical_story_summary_dict"?

Reply:
Done.